internal/cli/log_streams.go

```diff
@@ -289,7 +289,11 @@ func (c *cli) logStreamPickerOptionsByType(desiredType logStreamType) pickerOpti
 			}
 		}
 		if len(options) == 0 {
-			return nil, fmt.Errorf("there are currently no log streams of type: %s", desiredType)
+			return nil, fmt.Errorf(
+				"there are currently no log streams of type: %q, use 'auth0 logs streams create %s' to create one",
+				desiredType,
+				desiredType,
+			)
```

Comment:
We always give command hints in other places, so this just makes things consistent.